jdaviz/configs/default/plugins/virtual_observatory/vo_plugin.vue

```diff
@@ -0,0 +1,168 @@
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+    <j-plugin-section-header>Common Options</j-plugin-section-header>
+
+      <v-row justify="space-between">
+        <div :style="{ width: '55%' }">
+          <v-text-field
+            v-model.number="radius_val"
+            type="number"
+            label="Radius"
+            hint="Angular radius around source coordinates, within which to query for data (Default 1 degree)"
+            persistent-hint>
+          </v-text-field>
+        </div>
+        <div :style="{ width: '40%' }">
+          <v-select
+          v-model="radius_unit_selected"
+          attach
+          :items="radius_unit_items.map(i => i.label)"
+          label="Unit">
+          </v-select>
+        </div>
+      </v-row>
+
+
+    <j-plugin-section-header>Survey Collections</j-plugin-section-header>
+      <v-row>
+        <j-tooltip tipid='plugin-vo-filter-coverage'>
+        <span>
+          <v-btn icon @click.stop="resource_filter_coverage = !resource_filter_coverage">
+            <v-icon>mdi-filter{{ resource_filter_coverage ? '' : '-remove' }}</v-icon>
+          </v-btn>
+          Filter by Coverage
+        </span>
+        </j-tooltip>
+      </v-row>
+      
+      <v-row>
+        <v-select
+          v-model="waveband_selected"
+          :menu-props="{ left: true }"
+          attach
+          :items="wavebands"
+          label="Resource Waveband"
+          hint="Select a spectral waveband to filter your surveys"
+          persistent-hint
+        ></v-select>
+      </v-row>
+
+      <v-row>
+        <div :style="'width: calc(100% - 32px)'">
+          <v-select
+            v-model="resource_selected"
+            :menu-props="{ left: true }"
+            attach
+            :items="resources"
+            :loading="resources_loading"
+            :rules="[() => !!resource_selected || 'This field is required']"
+            label="Available Resources"
+            hint="Select a SIA resource to query"
+            persistent-hint
+          ></v-select>
+        </div>
+        <div style="line-height: 64px; width:32px">
+          <v-btn
+            id="querybtn"
+            @click="query_registry_resources"
+            icon
+            :loading="resources_loading">
+            <v-icon>mdi-refresh</v-icon>
+          </v-btn>
+        </div>
+      </v-row>
+    </v-form>
+
+    <v-row class="row-no-outside-padding">
+      <v-col>
+        <v-btn
+          block
+          color="primary"
+          :loading="results_loading"
+          :disabled="!all_fields_filled"
+          text
+          @click="query_resource">Query Archive</v-btn>
+      </v-col>
+    </v-row>
+
+    <jupyter-widget :widget="table_widget"></jupyter-widget>
+
+    <v-row class="row-no-outside-padding">
+        <v-col>
+            <v-btn color="primary" :loading="data_loading" text @click="load_selected_data">Load Data</v-btn>
+        </v-col>
+    </v-row>
```

Comment:
can these use `plugin-action-button` which should also provide consistent styling/layout (right now the first button takes the full width due to `block` while the second does not and is right-aligned)?

Reply:
Done in https://github.com/spacetelescope/jdaviz/pull/2872/commits/354cfd7b447d37583ec015bba7b84efced228ce2